tox.ini

```diff
@@ -10,10 +11,11 @@ setenv =
     PYTHONPATH = {toxinidir}
     DJANGO_SETTINGS_MODULE = wagtail_storages.tests.settings
 deps =
-    dj22: Django>=2.2,<2.3
-    dj30: Django>=3.0,<3.1
-    wagtail28: wagtail>=2.8,<2.9
-    wagtailmaster: git+https://github.com/wagtail/wagtail.git@master#egg=Wagtail
+    dj3: Django>=3,<4
+    dj4: Django>=4,<5
```

Comment:
wanted to go that is an awfully wide range, but then I see doing `dj40` seems to limit to Django 4.0.x
TIL!

Reply:
hmm... with `py310-dj4-wagtail{216,main}`, and 4.1 coming out very soon that range becomes too wide.

perhaps bite the bullet and do the more granular breakdown? you don't have to, mind you :)

Reply:
Wagtail pins the Django versions explicitly. I assume that should limit it.

Reply:
Thinking about it, if it does break, then we don't have an easy way to see that it broke in that particular version...